package.json

```diff
@@ -43,12 +49,14 @@
   },
   "devDependencies": {
     "@aws-cdk/assert": "^2.68.0",
+    "@aws-cdk/integ-tests-alpha": "2.154.1-alpha.0",
     "@commitlint/config-conventional": "^18.6.3",
     "@mrgrain/jsii-struct-builder": "^0.7.42",
     "@types/jest": "^29.5.12",
     "@types/node": "^18",
     "@typescript-eslint/eslint-plugin": "^7",
     "@typescript-eslint/parser": "^7",
+    "aws-cdk": "^2",
```

Comment:
Might the dependency tree take care of this?

```suggestion
```

https://www.npmjs.com/package/@aws-cdk/integ-tests-alpha?activeTab=dependencies
https://www.npmjs.com/package/@aws-cdk/integ-runner?activeTab=dependencies

Reply:
FYI: above is non-blocking

Reply:
I apologized and assumed incorrectly, looks like the build automagically adds `"aws-cdk: "^2",` it back in. Looks like https://github.com/awslabs/generative-ai-cdk-constructs/pull/663/commits/aafbd5489b7c836ab3b00b7b231c49610a70c1e4 can be reverted.

Reply:
@scottschreckengaust just reverted the commit, build is green